Modules/Chatroom/classes/class.ilChatroom.php

```diff
@@ -691,7 +693,10 @@ public function sendInvitationNotification(
             $notification->setIconPath('templates/default/images/icon_chtr.svg');
             $notification->setValidForSeconds(ilNotificationConfig::TTL_LONG);
             $notification->setVisibleForSeconds(ilNotificationConfig::DEFAULT_TTS);
-
+            $notification->setIdentification(new NotificationIdentification(
+                ChatInvitationNotificationProvider::NOTIFICATION_TYPE,
+                self::ROOM_INVITATION . '_' . $gui->getObject()->getRefId(),
```

Comment:
@mjansenDatabay I just removed the subscope here, as far as i remember right its removed for ilias 9, right?

Reply:
Exactly